*** kernel.adb	2011-03-11 12:43:34.000000000 -0600
--- kernel.GBA.adb	2011-03-18 16:38:11.000000000 -0500
***************
*** 14,24 ****
  --    GNU General Public License for more details.                           --
  --                                                                           --
  --    You should have received a copy of the GNU General Public License      --
  --    along with the Sly Kernel.  If not, see <http://www.gnu.org/licenses/>.--
  -------------------------------------------------------------------------------
! 
  package body Kernel is
  procedure Kernel_Main(magic : in Integer) is
     begin
        null;
     end Kernel_Main;
--- 14,24 ----
  --    GNU General Public License for more details.                           --
  --                                                                           --
  --    You should have received a copy of the GNU General Public License      --
  --    along with the Sly Kernel.  If not, see <http://www.gnu.org/licenses/>.--
  -------------------------------------------------------------------------------
! with GBAIO; use GBAIO;
  package body Kernel is
  procedure Kernel_Main(magic : in Integer) is
     begin
        null;
     end Kernel_Main;
